spec/features/dashboard_feature_spec.rb

```diff
@@ -27,6 +27,7 @@
       visit authenticated_root_path
 
       el = find("#download-kubeconfig")
+      wait_until { !find("#download-kubeconfig")[:disabled] }
```

Comment:
@vitoravelino max wait time applies to pretty much every Capybara finder and matcher, except for `first` and `all` (unless you specify one of the count options which forces them into waiting mode as well)